ebos/eclproblem.hh

```diff
@@ -2530,12 +2526,6 @@ private:
         // using message passing.
         this->model().syncOverlap();
 
-        // this is a hack to preserve the initial fluid states
-        initialFluidStates_ = tmpInitialFs;
-        // make sure that the stuff which needs to be done at the beginning of an episode
-        // is run.
-        this->beginEpisode();
-
```

Comment:
@andlaus Since the restart is done before the simulator is ready I had to remove beginEpisode() here. For the Flow wellModel i just added the beginEpisode step in the restart handling flow. Could this cause problem for the simple well model? I have tested the code with Ebos and restart and it works perfectly.

Reply:
As far as I know, the simple well model does not care. (maybe I'm wrong here, though. you can test this using `ebos_plain` on SPE1.)

But I'm not sure if not calling `beginEpisode()` on a restart is the semantically right thing to do. There can be arguments for both positions.

Reply:
I will be called by the simulator after the solution is initialized.

Reply:
okay, maybe this just an artifact...